configure_cmake.sh

```diff
@@ -109,6 +109,10 @@ Optional Packages:
                             libuuid include directory
     --with-uuid-libraries=DIR
                             libuuid library directory
+    --with-tirpc-includes=DIR
```

Comment:
We don't need this level of support.  I think it's fine to let it rely on pkg-config or normal system environment.